packages/front-end/components/Settings/InviteModal.tsx

```diff
@@ -23,6 +28,28 @@ const InviteModal: FC<{ mutate: () => void; close: () => void }> = ({
   const [emailSent, setEmailSent] = useState<boolean | null>(null);
   const [inviteUrl, setInviteUrl] = useState("");
   const { apiCall } = useAuth();
+  const { data } = useApi<SettingsApiResponse>(`/organization`);
+  const {
+    seatsInFreeTier,
+    pricePerSeat,
+    discountedPricePerSeat,
+  } = useStripeSubscription();
+
+  const activeAndInvitedUsers =
+    data.organization.members.length + data.organization.invites.length;
+
+  const currentPaidSeats = data.organization.subscription.qty || 0;
+
+  const hasActiveSubscription =
+    data.organization.subscription?.status === "active" ||
+    data.organization.subscription?.status === "trialing";
+
+  const canInviteUser = Boolean(
+    emailSent === null &&
+      (activeAndInvitedUsers < seatsInFreeTier ||
+        (currentPaidSeats >= seatsInFreeTier && hasActiveSubscription) ||
```

Comment:
This logic might need to change slightly. If someone's account is past_due and they go to add a new user, that will block them from being able to invite people. I think I recall you saying you didn't want that to happen.

Reply:
Update the logic so we don't block someone from inviting new users if they have a subscription so long as the status isn't cancelled. We don't want to block someone from doing something since the user is likely not the person paying (they need to send it to their billing dept, etc).